modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -243,24 +258,58 @@ If successful, the call returns an object containing all current settings for th
 == XDCR Advanced Settings
 
 The advanced settings for XDCR are as follows.
-Note that the subset of advanced settings that can be configured by means of Couchbase Web Console is provided at xref:xdcr-reference:xdcr-advanced-settings.adoc[XDCR Advanced Settings].
+The subset of advanced settings that can be configured by means of Couchbase Web Console is provided at xref:xdcr-reference:xdcr-advanced-settings.adoc[XDCR Advanced Settings].
 
 .XDCR Advanced Settings
 [#table_xdcr_settings,cols="140,80,200"]
 |===
 | Parameter | Value | Description
 
 | `checkpointInterval`
-| Integer (60 to 14400).
-| Default: 600.
+| Integer (60 to 14400)
+| Default: 600
 The interval for checkpointing in seconds.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `colMappingRules`
+| JSON Object
+| Collection-related rules according to which explicit mapping or migration is defined for the replication.
+See xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-explicit-mappings[Rules for Explicit Mappings] and xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-migration[Rules for Migration].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsExplicitMapping`
+| Boolean (true or false)
+| Whether the replication uses explicit mapping: see xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#rules-for-explicit-mappings[Rules for Explicit Mappings].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsMigrationMode`
+| Boolean (true or false)
+| Whether the replication uses migration mode: see xref:learn:clusters-and-availability/xdcr-with-scopes-and-collections.adoc#migration[Migration].
+
+This setting can only be established for and retrieved from an individual replication: it cannot be established or retrieved as part of global settings.
+
+| `collectionsMirroringMode`
```

Comment:
The mirroring mode is not something that is publicized since this was a feature that never made it into 7.0 or so.
The description is incorrect anyway so better to just remove this section

Reply:
Ok, I have removed this entry. My concern here would be that, since collectionsMIrroringMode currently does appear in the output, eventually, a ticket will eventually be filed, requesting that it be explained. So, will collectionsMIrroringMode in fact be removed from the output? If so, I can update the output accordingly.